__tests__/app.test.js

```diff
@@ -72,3 +72,39 @@ describe('GET /api/articles/:article_id', () => {
         })
     })
 })
+
+
+describe('GET /api/articles/:article_id/comments', () => {
+    test('200: responds with an array of comments from the corresponding article_id, with each comment containing the correct properties', () => {
+        return request(app).get('/api/articles/5/comments').expect(200).then(({ body }) => {
+            expect(body.comments).toHaveLength(2);
+            const testComment = {
+                comment_id: expect.any(Number),
+                body: expect.any(String),
+                article_id: expect.any(Number),
+                author: expect.any(String),
+                votes: expect.any(Number),
+                created_at: expect.any(String),
+            }
+            body.comments.forEach((comment) => {
+                expect(comment).toMatchObject(testComment);
+            })
+        })
+    })
+    test('200: should respond with the most recent comment first', () => {
+        return request(app).get('/api/articles/5/comments').expect(200).then(({ body }) => {
+            expect(body.comments).toBeSortedBy('created_at', { descending: true });
+        })
+    })
+    test('404: responds with an appropriate status and error message when given a valid id that does not exist', () => {
+        return request(app).get('/api/articles/9999/comments').expect(404).then(({ body }) => {
+            expect(body.msg).toBe('Comment does not exist!');
+
+        })
+    })
+    test('400: responds with an appropriate status and error message when given an invalid id', () => {
+        return request(app).get('/api/articles/not-valid/comments').expect(400).then(({ body }) => {
+            expect(body.msg).toBe('Invalid article ID');
+        })
+    })
+})
```

Comment:
One test missing here:
It is possible for an article to have no associated comments (there are examples of this in your data set). In this case we would expect status 200 and to recieve an empty array